style.css

```diff
@@ -1,294 +1,283 @@
-*{
-    box-sizing: border-box;
-}
-body{
-    background-color:#F7F8F8;
-}
-.container{
-    background-color: white;
-}
-
-html{
-    font-family: 'Poppins', sans-serif;
-}
-body{
-    padding-bottom:50px;
-}
-.header{
-    display: inline-block;
-}
-.row{
-    width:auto;
-    margin:0 10px;
-}
-.section-head{
-    background-color: #1D3B53;
-    color:white;
-    height: 47rem;
-
-} 
-
-.header-title{
-    text-align: right;
-    padding: 30px;
-    width: 80%;
-}
-
-.header-text{
-    text-align:left;
-    padding-top: 25%;
-}
-.dropdown-button{
-    padding:10px;
-    background-color:#1D3B53;
-    border: 0;
-    color:white;
-    width: 150%;
-    font-size: 1rem;
-    cursor: pointer;
-}
-
-.dropdown{
-    position: relative;
-    display: inline-block
-}
-
-.menu-item{
-    display: none;
-    position: absolute;
-    background-color: #f1f1f1;
-    min-width: 160px;
-    width:150%;
-    overflow: auto;
-    box-shadow: 0px 8px 16px 0px rgba(0,0,0,0.2);
-    text-align:center;
-    z-index: 1;
-}
-.navBar{
-    margin-bottom:-70px;
-}
-.navBar-linkText{
-    position:relative;
-    display: inline-block;
-    text-align: left;
-}
-.navBar-link:hover{
-    color:#FF102D;
-}
-.navBar-link{
-    position:relative;
-    text-decoration:none;
-    color:white;
-    background-color:transparent;
-    border:none;
-    width: 150%;
-    overflow:auto;
-    padding:10px 0;
-    font-size: 1rem;
-}
-.linkText-expert{
-    margin-left:-30px;
-}
-.navBar-header{
-    display:inline-block;
-    margin-top:-25px;
-}
-.menu-item li{
-    color: black;
-    padding: 12px 16px;
-    display: block;
-}
-
-.dropdown li:hover{
-    background-color: #ddd;
-    width:100px;
-    
-}
-
-.show{
-    display:block;
-}
-.redPunctuation{
-    color:#FF102D;
-}
-.button-round{
-    border: none;
-    height:3rem;
-    width:80%;
+* {
+	box-sizing: border-box;
+}
+body {
+	background-color: #F7F8F8;
+}
+.container {
+	background-color: white;
+}
+html {
+	font-family: 'Poppins', sans-serif;
+}
+body {
+	padding-bottom: 50px;
+}
+.header {
+	display: inline-block;
+}
+.row {
+	width: auto;
+	margin: 0 10px;
+}
+.section-head {
+	background-color: #1D3B53;
+	color: white;
+	height: 47rem;
+}
+.header-title {
+	text-align: right;
+	padding: 30px;
+	width: 80%;
+}
+.header-text {
+	text-align: left;
+	padding-top: 25%;
+}
+.dropdown-button {
+	padding: 10px;
+	background-color: #1D3B53;
+	border: 0;
+	color: white;
```

Comment:
Please use rgb or hex values

Reply:
Gone through and replaced all none hex values